src/test/java/com/datadog/api/TestUtils.java

```diff
@@ -286,6 +296,19 @@ public void setupClock() throws IOException {
             }
         }
 
+        @Before
+        public void setTracingTags() {
+            final Span span = GlobalTracer.get().activeSpan();
+            if (span != null) {
+                // if the agent container is not running, span is null
+                MutableSpan localRootSpan = ((MutableSpan) span).getLocalRootSpan();
+                localRootSpan.setTag(TRACING_TAG_ENDPOINT, getTracingEndpoint());
+                // we need to set both resourceName and the `resource.name` tag, which is what's displayed in the UI
+                localRootSpan.setResourceName(getQualifiedTestcaseName());
+                localRootSpan.setTag("resource.name", getQualifiedTestcaseName());
```

Comment:
can you also change the span type to `test`?

```golang
tracer.SpanType("test")
```

Reply:
sure 👍

Reply:
done